build.gradle

```diff
@@ -98,9 +98,9 @@ def scripts = [
     'DirectSubscriber':'com.solace.samples.jcsmp.patterns.DirectSubscriber',
     'GuaranteedProcessor':'com.solace.samples.jcsmp.patterns.GuaranteedProcessor',
     'GuaranteedPublisher':'com.solace.samples.jcsmp.patterns.GuaranteedPublisher',
-    'GuaranteedPublisherAckTrack':'com.solace.samples.jcsmp.patterns.GuaranteedPublisherAckTrack',
-    'GuaranteedReplier':'com.solace.samples.jcsmp.patterns.GuaranteedReplier',
-    'GuaranteedRequestor':'com.solace.samples.jcsmp.patterns.GuaranteedRequestor',
+//    'GuaranteedPublisherAckTrack':'com.solace.samples.jcsmp.patterns.GuaranteedPublisherAckTrack',
```

Comment:
Are these just placeholders until those new pattern actor sample exist?

Reply:
For some reason, these scripts were in the existing build file, even though they are not implemented.  I've removed for now, rather than commented out.